lib/bn/bn128_arithmetic.ex

```diff
@@ -1,57 +1,81 @@
 defmodule BN.BN128Arithmetic do
   require Integer
-  alias BN.IntegerModP
-  alias BN.IntegerModP.Point
+  alias BN.{FQ, FQP, FQ2, FQ12}
 
   # y^2 = x^3 + 3
   @y_power 2
   @x_power 3
-  @default_b IntegerModP.new(3)
+  @b FQ.new(3)
+  @b2 [3, 0] |> FQ2.new() |> FQ2.divide(FQ2.new([9, 1]))
```

Comment:
Not sure if this makes more sense without a pipeline. Purely stylistic, so not sure if it makes sense.